deploy/kubernetes/stable/controller.yaml

```diff
@@ -17,10 +17,10 @@ spec:
       containers:
         - name: csi-provisioner
           imagePullPolicy: Always
-          image: quay.io/k8scsi/csi-provisioner:v0.2.0
+          image: quay.io/k8scsi/csi-provisioner:v0.4.1
```

Comment:
can you open up a new issue to track using gke officially released public images for sidecars and driver? (the staging ones are not publicly accessible)

Reply:
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/issues/135